modules/terraform-ibm-transit-gateway-action/README.md

```diff
@@ -17,7 +17,7 @@ provider "ibm" {
 
 module "tg_gateway_connection_crossaccounts_approve" {
 
-  source  = "terraform-ibm-modules/transit-gateway/ibm"
+  source  = "terraform-ibm-modules/modules/transit-gateway/ibm"
```

Comment:
that is an incorrect source for the submodule. It should be `terraform-ibm-modules/transit-gateway/ibm//modules/terraform-ibm-transit-gateway-action`